client/lib/desktop/index.js

```diff
@@ -40,12 +43,31 @@ const Desktop = {
 		ipc.on( 'signout', this.onSignout.bind( this ) );
 		ipc.on( 'toggle-notification-bar', this.onToggleNotifications.bind( this ) );
 		ipc.on( 'page-help', this.onShowHelp.bind( this ) );
+		ipc.on( 'enable-site-option', this.desktopCommandEnableSiteOption.bind( this ) );
```

Comment:
Is it possible to only add listeners when they are needed and remove them afterwards?

Reply:
Yep - refactored accordingly.